### PR TITLE
[NUI] Fix build warnings

### PR DESCRIPTION
**Pull request by dongsug-song**
### Description of Change ###
[NUI] Fix build warnings

### API Changes ###
http://suprem.sec.samsung.net/jira/browse/TCSACR-248

### CODE REVIEW

**Comment by dongsug-song** [comment]
@huiyueun 
@adeelkazmi 
@AdunFang 

Hello, please review this. thank you.

File: src/Tizen.NUI/src/public/Accessibility.cs [hunk 1]
```diff
@@ -38,6 +38,9 @@ private Accessibility()
             dummy = new View();
             dummy.Name = "dali-atspi-singleton";
         }
+        /// <summary>
+        /// distructor
```
Comment (m-macias) [inline src/Tizen.NUI/src/public/Accessibility.cs]:
```suggestion
        /// destructor
```
Redundant comment.

File: src/Tizen.NUI.Components/Controls/FlexibleView/LinearLayoutManager.cs [hunk 2]
```diff
@@ -947,6 +947,10 @@ private void UpdateLayoutStateToFillStart(int itemPosition, float offset)
             mLayoutState.Extra = mOrientationHelper.GetStartAfterPadding();
         }
 
+        /// <summary>
+        /// FindFirstVisibleItemView
+        /// </summary>
+        /// <returns></returns>
```
Comment (m-macias) [inline src/Tizen.NUI.Components/Controls/FlexibleView/LinearLayoutManager.cs]: Empty return tag.

File: src/Tizen.NUI.Components/Controls/FlexibleView/LinearLayoutManager.cs [hunk 3]
```diff
@@ -1007,6 +1011,10 @@ private FlexibleView.ViewHolder FindFirstCompleteVisibleItemView()
             return null;
         }
 
+        /// <summary>
+        /// FindLastVisibleItemView
+        /// </summary>
+        /// <returns></returns>
```
Comment (m-macias) [inline src/Tizen.NUI.Components/Controls/FlexibleView/LinearLayoutManager.cs]: Empty return tag.

File: src/Tizen.NUI.Components/Controls/FlexibleView/FlexibleView.cs [hunk 4]
```diff
@@ -1946,11 +1946,19 @@ private void AddViewInternal(ViewHolder holder, int index, bool disappearing)
                 }
             }
 
+            /// <summary>
+            /// FindFirstVisibleItemView
+            /// </summary>
+            /// <returns></returns>
```
Comment (m-macias) [inline src/Tizen.NUI.Components/Controls/FlexibleView/FlexibleView.cs]: Empty return tag.

File: src/Tizen.NUI.Components/Controls/FlexibleView/FlexibleView.cs [hunk 5]
```diff
@@ -1946,11 +1946,19 @@ private void AddViewInternal(ViewHolder holder, int index, bool disappearing)
...
             protected virtual ViewHolder FindFirstVisibleItemView()
             {
                 return null;
             }
 
+            /// <summary>
+            /// FindLastVisibleItemView
+            /// </summary>
+            /// <returns></returns>
```
Comment (m-macias) [inline src/Tizen.NUI.Components/Controls/FlexibleView/FlexibleView.cs]: Empty return tag.

File: src/Tizen.NUI.Components/Controls/Button.cs [hunk 6]
```diff
@@ -756,10 +757,11 @@ public Extents IconPadding
             }
         }
 
-        // <summary>
+        /// <summary>
         /// Text padding in Button, work only when show icon and text.
         /// </summary>
         /// <since_tizen> 6 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
```
Comment (m-macias) [inline src/Tizen.NUI.Components/Controls/Button.cs]: Hiding public API.

**Pull request merged**